api/core/model_runtime/model_providers/localai/llm/llm.py

```diff
@@ -316,7 +316,7 @@ def _to_client_kwargs(self, credentials: dict) -> dict:
         client_kwargs = {
             "timeout": Timeout(315.0, read=300.0, write=10.0, connect=5.0),
             "api_key": "1",
-            "base_url": join(credentials['server_url'], 'v1'),
+            "base_url": urljoin(credentials['server_url'], 'v1'),
```

Comment:
Could you pls help add this part to avoid an issue where the last segment of the subpath is replaced by `v1` due to `server_url` not ending with `/`?

```
if not credentials['server_url'].endswith('/'):
    credentials['server_url'] += '/'
```

Thank you very much!

Reply:
okay